package.json

```diff
@@ -90,10 +90,10 @@
     "astro-pagefind": "^1.8.3",
     "astro-seo": "^0.7.5",
     "bundlesize": "^1.0.0-beta.2",
-    "cem-plugin-vs-code-custom-data-generator": "^1.4.2",
```

Comment:
This packages was deprecated and replaced by `custom-element-vs-code-integration`